api/src/main/java/org/openmrs/module/xdssender/api/service/CcdService.java

```diff
@@ -11,4 +11,6 @@
 	Ccd getLocallyStoredCcd(Patient patient);
 	
 	Ccd downloadAndSaveCcd(Patient patient) throws XDSException, IOException;
+	
+	void downloadCcdAsPDF(Patient patient);
```

Comment:
this method should take an inputstream or a writer as a param for outputting the pdf

Reply:
You're right